crates/oxc_ast/src/ast/js.rs

```diff
@@ -148,6 +148,11 @@ impl<'a> Expression<'a> {
         matches!(self, Self::NumberLiteral(lit) if lit.value == 0.0)
     }
 
+    /// Determines whether the given literal is exactly "val"
+    pub fn is_raw(&self, val: &str) -> bool {
```

Comment:
sorry, can we rename ths `is_specific_raw_number_literal`. since this is on `Expression`, and only matches `NumberLiteral`, the name feels a bit to broad